### PR TITLE
Ensure selection is displayed on top of Page navigation arrows

### LINKED ISSUE #2814: Selection handlers should take precedence over page navigation arrows
## Bug Description

Currently, when selection handlers are at the same location as the page navigation and the handler is clicked, the navigation action is triggered.

## Expected Behaviour
Selection handlers should take precedence over page navigation arrows

## Steps to Reproduce
1. Add an element
2. Resize it so that its resize handlers are in the same place as the page navigation
3. Try resizing now
4. See the page navigating instead

## Additional Context
Follow-up to https://github.com/google/web-stories-wp/issues/2463, see also https://github.com/google/web-stories-wp/issues/2463#issuecomment-651275296

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## Summary
Moves the page nav to the same level with page menu and selection so that it could be layered as expected.

## User-facing changes
- Selection now appears on top of the page navigation arrows.

## Testing Instructions
1. Add a new Page
2. Add an element
3. Resize and move the element so that the rotation handle would be on top of the page navigation arrow
4. Verify that the rotation handle is on top of the arrow and can still be used

---

Fixes #2814

### CODE REVIEW

File: assets/src/edit-story/components/canvas/karma/selection.karma.js [hunk 1]
```diff
@@ -87,7 +87,31 @@ describe('Selection integration', () => {
...
+      fixture.screen.getByRole('button', { name: 'Previous Page' })
+    );
+    const frame1 = fixture.editor.canvas.framesLayer.frame(element1.id).node;
+    await fixture.events.click(frame1);
+    const fbcr = frame1.getBoundingClientRect();
+    await fixture.events.mouse.seq(({ moveRel, moveBy, down, up }) => [
+      moveRel(frame1, 10, 10),
+      down(),
+      moveBy(
+        fullbleed.getBoundingClientRect().width - 20,
```
Comment (miina) [inline assets/src/edit-story/components/canvas/karma/selection.karma.js]: Note -- still need to fix this to position it correctly based on the position of the nav arrow.

File: assets/src/edit-story/components/canvas/pagemenu/index.js [hunk 2]
```diff
@@ -180,7 +180,7 @@ function PageMenu() {
   }
 
   return (
-    <Wrapper>
+    <Wrapper role="list" aria-label={__('Page actions', 'web-stories')}>
```
Comment (barklund) [inline assets/src/edit-story/components/canvas/pagemenu/index.js]: Shouldn't all intermediate elements be `role="presentation"` then or? I see 3-4 React components in-between but don't know how many actual DOM nodes there are.

File: assets/src/edit-story/components/canvas/pagemenu/index.js [hunk 3]
```diff
@@ -90,7 +90,7 @@ const Space = styled.div`
   width: ${({ isDouble }) => (isDouble ? 20 : 10)}px;
 `;
 
-const Icon = styled.button`
+const Icon = styled.button.attrs({ role: 'listitem' })`
```
Comment (barklund) [inline assets/src/edit-story/components/canvas/pagemenu/index.js]: Well, then they're not "real" buttons anymore? Can you still click them using the keyboard (space or enter) when focused? Does it have any other potential drawbacks being not-buttons? Also, why are they not real buttons anymore?

**Review by barklund — APPROVED** [review]
This works great, but not sure about the `role="list"` change - what's the purpose/motivation behind it?

File: assets/src/edit-story/components/canvas/pagemenu/index.js (hunk 2 above)
Comment (miina) [reply assets/src/edit-story/components/canvas/pagemenu/index.js]: Oh, this was initially mainly for testing purposes, reverted the change.

**Review by barklund — APPROVED** [review]

File: assets/src/edit-story/components/canvas/pagemenu/index.js (hunk 2 above)
Comment (barklund) [reply assets/src/edit-story/components/canvas/pagemenu/index.js]: Ah, if you want it to be a "targetable" group element, then you should probably just make it a `role="region"` with `<section />` (or `role="complementary"` with `<aside />` even if you're feeling fancy).

**Comment by csossi** [comment]
Verified in QA

**Pull request merged**